### PR TITLE
Added a target to clean OMTLMSimulator from OMSimulator Makefile

### PR DESCRIPTION
**Pull request by adeas31**
## Purpose

Allow cleaning the OMTLMSimulator build from OMSimulator makefile

## Approach

Added new target `OMTLMSimulatorClean`

## Type of Change

- New feature (non-breaking change which adds functionality)

## Checklist

- I have performed a self-review of my own code

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -88,6 +88,12 @@ OMTLMSimulatorStandalone: config-fmil
 	@echo $(ABI)
 	@$(MAKE) -C OMTLMSimulator install
 
+OMTLMSimulatorClean:
+	@echo
+	@echo "# clean OMTLMSimulator"
+	@echo
+	@echo $(ABI)
```
Comment (robbr48) [inline Makefile]: It is not really necessary to print the ABI variable before cleaning.

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/406//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#226

The changes will be pushed if all tests run successfully.

File: Makefile [hunk 2]
```diff
@@ -88,6 +88,11 @@ OMTLMSimulatorStandalone: config-fmil
 	@echo $(ABI)
 	@$(MAKE) -C OMTLMSimulator install
 
+OMTLMSimulatorClean:
```
Comment (lochel) [inline Makefile]: I guess this target should be added tp `.PHONY`.

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/407//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#226

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
All tests were successful. :smile:

**Pull request closed without merging**